doc/api/http2.md

```diff
@@ -1505,8 +1505,7 @@ added: v8.4.0
 * `options` {Object}
   * `allowHTTP1` {boolean} Incoming client connections that do not support
     HTTP/2 will be downgraded to HTTP/1.x when set to `true`. The default value
-    is `false`. See the [`'unknownProtocol'`][] event. See [ALPN
-    negotiation](#alpn-negotiation).
+    is `false`. See the [`'unknownProtocol'`][] event. See [ALPN negotiation][].
```

Comment:
This line is now 81 chars long. I suggest:

    See the [`'unknownProtocol'`][] event. See [ALPN negotiation][].

Reply:
It seems it is exactly 80 chars currently.